services/contributions.js

```diff
@@ -45,6 +48,7 @@ const getUserContributions = async (username) => {
       }
 
       noteworthyObject.task.participants = participantsDetails;
+      collapsedTaskObject.task.participants = participantsDetails;
```

Comment:
@sahsisunny do we need to store these details, as I have seen in website-members PR, we don't show details, if the task is collapsed

Reply:
@sahsisunny any update on this?

Reply:
Hello Vikhyat,  Right now I am paying full attention to an assignment(JM) so that's why I have put it on hold, once it is cleared I will start working on it again.

Reply:
> @sahsisunny do we need to store these details, as I have seen in website-members PR, we don't show details, if the task is collapsed

not @vikhyat187 , i thought i have to collapse the PR but it's not possible without storing(PR Details) in rds backend